components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -105,6 +106,23 @@ - (MDCBaseTextFieldLayout *)calculateLayoutWithTextFieldSize:(CGSize)textFieldSi
                                                      isEditing:self.isEditing];
 }
 
+/**
+To understand this method one must understand that the CGRect UITextField returns from @c
+-textRectForBounds: does not actually represent the CGRect of visible text in UITextField. It
+represents the CGRect of an internal "field editing" class, which has a height that is significantly
+taller than the text (@c font.lineHeight) itself. Providing a height in @c -textRectForBounds: that
+differs from the height determined by the superclass results in an unusable text field. By taking
```

Comment:
What does "unusable text field" mean in this context? Please expand on it here in the docs.